packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -10,13 +10,42 @@ location /pkgpanda/active.buildinfo.full.json {
 
 # Group: Root
 # Description: DC/OS GUI
-root /opt/mesosphere/active/dcos-ui/usr;
+# root /opt/mesosphere/active/dcos-ui/usr;
 
 location / {
     # prevent opening in iframe
     add_header X-Frame-Options DENY;
 }
 
+# Group: DC/OS UI
+# Description: the exact match / to the UI Service
+location = / {
```

Comment:
we should be able to have this as very last entry in this config and use `^/(.*)$` wildcard to send everything that didnt match anything else to our service.

Reply:
I would like not to rely on the order